packages/shopping/public/config.js

```diff
@@ -5,7 +5,11 @@
 
 'use strict';
 
-const apiUrl = 'http://localhost:3000';
+const apiUrl =
```

Comment:
@dougal83 @hacksparrow @jannyHou what do you think?

Reply:
Oh I totally missed the point previously!  IMO just hard code in the case of the example but that looks like it will serve the purpose to make it dynamic.  Bear in mind the example is a bear bones demonstration, I don't think it was intended for production.

@hacksparrow is your man to give definitive answer. 👍

Reply:
@johanschvn I would recommend using `window.location.origin` instead

Reply:
@nflaig +1 on window.location.origin. I think apiUrl is more descriptive since this is on the frontend side.